pennylane/math/multi_dispatch.py

```diff
@@ -173,13 +173,13 @@ def diag(values, k=0):
     **Example**
 
     >>> x = [1., 2., tf.Variable(3.)]
-    >>> diag(x)
+    >>> qml.math.diag(x)

```

Comment:
Note: it might have been assumed here that we have `diag` in the scope already because it's defined in this file and that the import was assumed, but agree with the change as it's more easily reproducible when copied.